src/html/scope.rs

```diff
@@ -108,20 +108,28 @@ impl<COMP: Component> Scope<COMP> {
     }
 
     /// Send a message to the component
-    pub fn send_message(&self, msg: COMP::Message) {
-        self.update(ComponentUpdate::Message(msg));
+    pub fn send_message<T>(&self, msg: T)
+    where
+        T: Into<COMP::Message>,
+    {
+        self.update(ComponentUpdate::Message(msg.into()));
     }
-
     /// Send a batch of messages to the component
-    pub fn send_message_batch(&self, messages: Vec<COMP::Message>) {
-        self.update(ComponentUpdate::MessageBatch(messages));
+    pub fn send_message_batch<I>(&self, messages: I)
```

Comment:
On second thought, I don't think this change helps the API. It actually goes against the guidelines: https://rust-lang.github.io/api-guidelines/flexibility.html#caller-decides-where-to-copy-and-place-data-c-caller-control

Since we require ownership over the `Vec<COMP::Message>` we should have that be the method argument, otherwise we may unnecessarily allocate.

Please revert these changes and the changes to `callback_batch`. Sorry!

Reply:
So, should I keep `IntoIterator + Into`?
If no, could you please specify more precisely?

Reply:
I would like all changes in this PR reverted for `send_message_batch` and `callback_batch`